book.rb

```diff
@@ -0,0 +1,9 @@
+# book.rb
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] According to the project requirements, you are expected to have a method for adding a rental. Please add that so a rental for a book can be added. 👍🏽

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```